src/DFTK.jl

```diff
@@ -7,6 +7,7 @@ using Printf
 using TimerOutputs
 using Unitful
 using UnitfulAtomic
+using DiffResults
```

Comment:
If you only use it in one file, I prefer to put the `using` in that file only to keep it local to the usage.